test/PerformanceTests/SerializationComparisonsTests/JsonWriterBenchmarks/Benchmarks.cs

```diff
@@ -65,29 +71,30 @@ public async Task WriteToMemoryAsync()
         public async Task WriteToFileAsync()
         {
             // multiple writes to increase benchmark duration
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
-            await WritePayloadAsync();
+            await WritePayloadAsync(data);
+            await WritePayloadAsync(data);
         }
 
         [Benchmark]
         [BenchmarkCategory("RawValues")]
         public async Task WriteWithRawValues()
         {
             // multiple writes to increase benchmark duration

```

Comment:
nit: here to..

Reply:
See: https://github.com/OData/odata.net/pull/2849/files#r1472336559